### DISCUSSION TITLE
I am getting array error

### DISCUSSION THREAD

**Discussion by TheNewJosh** [ctwillie/expo-server-sdk-php#20] [Q&A]
Below is the error I am getting

TypeError
ExpoSDK\Utils::isAssoc(): Argument #1 ($arr) must be of type array, string given, called in /home/u555174784/domains/azcpay.com/public_html/core/vendor/ctwillie/expo-server-sdk-php/src/ExpoMessage.php on line 180

Looking forward to help on this channel

**Comment by ctwillie** [comment]
@TheNewJosh Hello! Can you share a code snippet that is producing this error?

**Reply by ctwillie** [reply]
@TheNewJosh Any update with this?

**Reply by ctwillie** [reply]
@TheNewJosh You must have passed a `string` to the `setData` method on the ExpoMessage instance. I will put in a fix for this in the next day or so. Or you can PR if you like! Thanks for reporting.
